nginx/Dockerfile

```diff
@@ -30,22 +48,9 @@ COPY src/opt/modsecurity/activate-rules.sh /opt/modsecurity/
 COPY src/etc/modsecurity.d/*.conf /etc/modsecurity.d/
 COPY nginx/docker-entrypoint.sh /
 COPY nginx/nginx.conf /etc/nginx/nginx.conf
+COPY --from=release /opt/owasp-crs /opt/owasp-crs
 
-# Change default shell to bash
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
I see this wasn't even in the NGiNX file. Did this even serve any purpose?

Reply:
The same that `set -eux` serves now: to fail whenever any command in the list fails.